scripts/generate-vpn-certs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This script generates new TLS server and client certs for 
+# mutual-auth AWS Client VPN.
+# Instructions based on: 
+# https://docs.aws.amazon.com/vpn/latest/clientvpn-admin/authentication-authorization.html#mutual
+# Make sure you have the EXISTING server and client cert ARN's passed in as the 
+# first and second arguments, respectively, or you will upload/generate new certificates
+# that aren't associated with the Client VPN endpoint.
+
+# Remove old dirs for a fresh start
+rm -rf ~/easy-rsa ~/vpn-certs
+
+cd ~/
+git clone https://github.com/OpenVPN/easy-rsa.git
```

Comment:
This is based on the [AWS directions](https://docs.aws.amazon.com/vpn/latest/clientvpn-admin/authentication-authorization.html#mutual). I'm guessing they want to keep latest release to maintain security updates merged to master instead of pinning to a release.

Reply:
Got it. Let's pin to a version in our script, since [using `master` is not recommended](https://github.com/OpenVPN/easy-rsa#branch-structure):

> Please note that, at any given time, master may be broken. Feel free to create issues against master, but have patience when using the master branch. It is recommended to use a release, and priority will be given to bugs identified in the most recent release.

Reply:
Oh dear gods, good job Amazon 🙄 . Will pin to v3.0.6